modules/nw-ovn-kubernetes-migration.adoc

```diff
@@ -289,6 +289,11 @@ daemon set "multus" successfully rolled out
 
 . To complete changing the network plugin, reboot each node in your cluster. You can reboot the nodes in your cluster with either of the following approaches:
 
+[IMPORTANT]
```

Comment:
I think you need a `+ here. The spacing is off in the preview
```suggestion
+
[IMPORTANT]
```
![image](https://github.com/openshift/openshift-docs/assets/25209155/0ebffb96-aa23-454c-959a-f763586fcb0c)

Reply:
Yikes, good catch, thanks.